app/src/main/java/com/duckduckgo/app/global/DuckDuckGoApplication.kt

```diff
@@ -135,18 +92,10 @@ open class DuckDuckGoApplication : HasAndroidInjector, Application(), LifecycleO
             }
         }
 
-        loadTrackerData()
-        scheduleOfflinePixels()
-
-        notificationRegistrar.registerApp()
-        registerReceiver(shortcutReceiver, IntentFilter(ShortcutBuilder.SHORTCUT_ADDED_ACTION))
-
-        initializeHttpsUpgrader()
         submitUnsentFirePixels()
 
         appCoroutineScope.launch {
             referralStateListener.initialiseReferralRetrieval()
```

Comment:
I could also move the referral bit into its own plugin but I am unsure whether this is executed last for a reason. So left it here for now